src/renderer/components/cluster-manager/catalog-topbar.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2021 OpenLens Authors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import React from "react";
+import { welcomeURL } from "../../../common/routes";
+import { navigate } from "../../navigation";
+import { Icon } from "../icon";
+import { TopBar } from "../layout/topbar";
+import { MaterialTooltip } from "../material-tooltip/material-tooltip";
+
+export function CatalogTopbar() {
+  return (
+    <TopBar label="Catalog">
+      <div>
+        <MaterialTooltip title="Close Catalog" placement="left">
+          <Icon style={{ cursor: "default" }} material="close" onClick={() => navigate(welcomeURL())}/>
```

Comment:
Why are we overriding the cursor here?

Reply:
Because draggable area on OSX hovers part of the topbar, preventing to show `cursor: pointer` on any element. But, it hovers only part of it so if you hover quickly over the button, you'll see blinking cursor. This fix prevents any blinking and shows always a default cursor here.

Reply:
Sounds good.

Reply:
Maybe add a comment.